.github/workflows/stubs.yml

```diff
@@ -41,6 +41,10 @@ jobs:
         python3 -m pip install -U pip setuptools wheel
         python3 -m pip install -U pip mpi4py pytest pybind11-stubgen pre-commit seleniumbase
 
+    - name: Dashboard Dependencies
+      run: |
+        python3 -m pip install -U trame trame-matplotlib trame-plotly trame-router trame-xterm trame-vuetify
```

Comment:
It seems that adding this caused stub files test to continuously run.

Reply:
fixed